lib/auth/backends.py

```diff
@@ -21,6 +21,8 @@
     # Django name:  LDAP name(s),
     'Localizers': ('scm_l10n', 'locale_peer'),
     'build': ('buildteam', 'shipit'),
+    'l10n-drivers': ('scm_l10n_drivers',),
+    'l10n-infra': ('scm_l10n_infra',)
```

Comment:
This breaks the tests, and would also break auth on prod, I'll back that out.

I want to get rid of one of the two anyway.